kdestandardize.ado

```diff
@@ -1387,7 +1400,7 @@ prog def kdestandardize
 									cond(equity_measure=="Overall Student Growth Rating of Teachers and Leaders","4", ///
 									cond(equity_measure=="Percentage of new and Kentucky Teacher Internship Program (KTIP) teachers","5", ///
 									cond(equity_measure=="Percentage of Teacher Turnover","6", ///
-									cond(equity_measure=="School Leadership Composite","7")))))) ///
+									cond(equity_measure=="School Leadership Composite","7"))))))) ///
```

Comment:
Still need to add the third function argument to this function call for it to execute without throwing an error.